estuary/api/v1.py

```diff
@@ -113,6 +113,7 @@ def _get_partial_story(results, reverse=False):
         rv['meta']['story_related_nodes_forward'] = [0]
         rv['meta']['story_related_nodes_backward'] = [0]
         rv['meta']['requested_node_index'] = 0
+        rv['meta']['story_manager'] = type(story_manager).__name__
```

Comment:
I'm torn about this because it is very convenient to just pass the class but at the same time, this is very much an implementation detail that shouldn't really be exposed in the API.